src/pages/createCostModelWizard/markup.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  FormGroup,
+  InputGroup,
+  InputGroupText,
+  TextInput,
+  Title,
+} from '@patternfly/react-core';
+import React from 'react';
+import { CostModelContext } from './context';
+
+const Markup = () => (
+  <CostModelContext.Consumer>
+    {({ onMarkupChange, markup }) => {
+      const isValidMarkup = !isNaN(Number(markup));
+      return (
+        <>
+          <Title size="xl">Set a markup</Title>
+          <Title size="md">
+            To apply a markup or discount, enter in a (posotive or negative)
+            precentage value to add or substract to the base cost of the
+            source(s).
+          </Title>
+          <FormGroup
+            label="Markup rate"
+            isRequired
+            fieldId="markup"
+            helperTextInvalid={'Markup should be a posotive or negative number'}
+            isValid={isValidMarkup}
+          >
+            <InputGroup>
+              <TextInput
+                isRequired
+                type="text"
+                id="markup"
+                name="markup"
+                value={markup}
+                onChange={onMarkupChange}
+                isValid={isValidMarkup}
+              />
+              <InputGroupText>%</InputGroupText>
```

Comment:
Similar issue as the filter icon. That is, the percentage appears outside the text field, surrounded by a border. The mockup shows something more like this:

<img width="147" alt="Screen Shot 2019-08-16 at 10 39 52 AM" src="https://user-images.githubusercontent.com/17481322/63175749-ab749780-c012-11e9-84cf-fb495bc35f52.png">